### PR TITLE
[IT-1186] Set up JumpCloud access for Nextflow Prod account

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
This PR builds on top of #130 and sets up access to the AWS account via JumpCloud. I don't expect to be granted admin access on this account. I'm thinking that Sage IT can log into the account to create the service user and role required for me to set up the provisioner repository. 

- [x] If I understand correctly, @zaro0508 or @ahayden needs to generate the metadata document and store it as `workflows-nextflow-prod-admin.xml`. 

----

### PR Checklist
- [x] Describe and explain your intentions for this change
- [x] Setup pre-commit and run the validators (info in README.md)
  To validate files run: `pre-commit run --all-files`

### CODE REVIEW

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -285,3 +286,15 @@ Organization:
         Project: iAtlas
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws-iatlas-prod@sagebase.org
+
+  WorkflowsNextflowProdAccount:
```
Comment (zaro0508) [inline org-formation/organization.yaml:290]: this is already in PR #135

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Comment by BrunoGrandePhD** [comment]
I've added the metadata document that @zaro0508 provided on Slack. I also merged the `master` branch into this PR.

**Comment by BrunoGrandePhD** [comment]
@ahayden: Do you want to track the manual steps for setting up this production account here or on [IT-1186](https://sagebionetworks.jira.com/browse/IT-1186)?

**Review by zaro0508 — APPROVED** [review]

**Comment by BrunoGrandePhD** [comment]
@zaro0508: I dealt with the merge conflict.

**Pull request merged**